python/lsst/ctrl/bps/report.py

```diff
@@ -46,80 +44,6 @@
 _LOG = logging.getLogger()
 
```

Comment:
the things in this file seem like they've been replaced by `cli/script/report.py`. Did you mean to remove this file? Or maybe you want to keep some of these definitions? Maybe it wants a different name? (I'm not familiar with the context here, so whatever you think is best)